plugins/generic/pln/classes/Deposit.inc.php

```diff
@@ -112,11 +179,35 @@ function setNewStatus() {
 	}
 
 	/**
+	 * Get a status from the bit field.
+	 * 
+	 * @param $field int one of the PLN_PLUGIN_DEPOSIT_STATUS_* constants.
+	 * @return int
+	 */
+	function _getStatusField($field) {
+		return $this->getStatus() & $field;
+	}
+	
+	/**
+	 * Set a status value.
+	 * 
+	 * @param boolean $value 
+	 * @param int $field one of the PLN_PLUGIN_DEPOSIT_STATUS_* constants.
+	 */
+	function _setStatusField($value, $field) {
+		if($value) {
```

Comment:
Status is stored as a bit field. If `$value` is true, it sets the corresponding bit to one, otherwise it sets it to false.